README.md

```diff
@@ -60,131 +80,50 @@ systemctl start docker
 
-Run as `root`:
+Here are the commands for starting the YaST container. If you want to use the
+other images than just replace `ncurses` with `qt` or `web`.
 
-```sh
-# add the YaST:Head repository
-zypper addrepo -f 'https://download.opensuse.org/repositories/YaST:/Head/openSUSE_Leap_${releasever}/' YaST:Head
-# install the package
-zypper install yast-in-container
-# disable the repository to not accidentally install some other YaST packages
-zypper modifyrepo -d YaST:Head
-# run the script (from /usr/sbin/ or /sbin)
-yast2_container repositories
-```
-
-```sh
-# download the scripts
-git clone https://github.com/yast/yast-in-container.git
-cd yast-in-container/src/scripts
-# run the script
-./yast2_container repositories
-```
-
-## Details
-
-This repository provides several  helper scripts
-
-- `yast_container` - runs YaST with the ncurses (text mode) UI
-- `yast2_container` - runs YaST with the Qt (graphical mode) UI
-- `yast2_web_container` - runs YaST inside Web session, open the
-  https://localhost:4984 URL in a web browser, see more
-  [details below](#web-https-access)
-
-The scripts download the container image from OBS and then run the specified
-YaST module in the container. If no YaST module is specified then the YaST
-control center is started.
-
-The container is automatically deleted after finishing YaST, if you want to
-inspect the container you have to start it manually.
-
-## Debugging
+### Podman
 
-If you need to run a different command than YaST in the container for testing
-purposes you can specify it with the `CMD` environment variable.
+Running in Podman is easy:
 
-```sh
-# run shell for an interactive session instead of running YaST
-CMD=bash yast2_container
+```shell
+podman container runlabel run registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/yast-mgmt-ncurses:latest
 ```
 
-The script prints the executed `podman` or `docker` command, if you need to
-run the container with different options just copy&paste the printed command
-to the terminal.
-
-## Supported Modules
-
-Not all YaST modules can be directly used from a container, usually they need
-some small adjustments to work properly.
-
-The YaST control center in the container displays only the supported modules.
-Alternatively you can run `yast2_container -l` command to list the supported
-modules.
-
-The container image contains some more modules which are not displayed in the
-control center. These modules have been installed by dependencies but they will
-probably not work properly, do not use them.
-
-## Implementation Details
-
-### Accessing the Host System
-
-The host system is mounted to `/mnt` directory in the container. YaST must use
-this subdirectory instead of the root directory for reading/writing files.
-
-But that's usually not supported, esp. for modules designed to run only in
-an installed system.
+If you want to see the command which will be executed in advance then use the
+`--display` option:
```

Comment:
I would encourage users to always do it, as it can run anything. So it is good practice to check at first what it will run.